src/Propel/Runtime/Map/DatabaseMap.php

```diff
@@ -93,7 +93,7 @@ public function addTableObject(TableMap $table)
         $table->setDatabaseMap($this);
         $this->tables[$table->getName()] = $table;
         $phpName = $table->getClassName();
-        if ('\\' !== $phpName[0]) {
+        if ('\\' !== ($phpName ? $phpName[0] : null)) {
```

Comment:
here it makes more sense to use

    if ($phpName && '\\' !== $phpName[0]) {

since `\` itself doesn't make sense.

Reply:
But then also the next assignment is weird.
Maybe return early instead?

Reply:
Can you use the propose changes feature? It would be faster.